httpevents/handler.go

```diff
@@ -71,3 +72,11 @@ func (w *responseWriter) WriteHeader(status int) {
 		w.ResponseWriter.WriteHeader(status)
 	}
 }
+
+func (w *responseWriter) Hijack() (conn net.Conn, rw *bufio.ReadWriter, err error) {
+	if conn, rw, err = w.ResponseWriter.(http.Hijacker).Hijack(); err == nil {
```

Comment:
It panics, which gets caught and handled by the http server. There's no good guidelines in the standard library on what the behavior should be for wrapper so I'm not sure what's best (maybe it should just return an error but then what? the HTTP handler probably just panics with the error or silence it which may be worse).